Source/Shaders/Builtin/Functions/vertexLogDepth.glsl

```diff
@@ -0,0 +1,49 @@
+#ifdef LOG_DEPTH
+varying float v_logZ;
```

Comment:
Here and in `writeLogDepth.glsl`, we generally don't add new varyings or uniforms when including a built-in GLSL function (well, I also just noticed `writeDepthClampedToFarPlane.glsl`).  Instead, we try to make the functions standalone and pass in / return anything needed.  Is that reasonable here?  Or does that require too much shader patching elsewhere?